windows/ReactTestApp/MainPage.cpp

```diff
@@ -29,65 +23,42 @@ namespace winrt::ReactTestApp::implementation
             newMenuItem.IsEnabled(false);
             menuItems.Append(newMenuItem);
         } else {
+            std::vector<ReactTestApp::ComponentViewModel> components;
```

Comment:
It seems like a waste to have a `vector` here and only use the first element below. Can we optimize this?